sdk/src/main/java/io/dapr/client/DaprClientHttpAdapter.java

```diff
@@ -192,21 +174,21 @@
       if (state.getEtag() != null && !state.getEtag().trim().isEmpty()) {
         headers.put(Constants.HEADER_HTTP_ETAG_ID, state.getEtag());
       }
-      String serializedKeyBody = objectSerializer.serialize(state.getKey());
+      String serializedKeyBody = state.getKey();
```

Comment:
if no need to serialize, do we really need this variable anymore?

Reply:
I will change it. I did a big refactoring without changing more than needed. Will update.